modules/gathering-clusterversion-history.adoc

```diff
@@ -0,0 +1,82 @@
+// Module included in the following assemblies:
+//
+// * updating/troubleshooting_updates/gathering-data-cluster-update.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gathering-clusterversion-history_{context}"]
+= Gathering ClusterVersion history 
+
+When updating your cluster, the cluster version operator (CVO) creates a `status.history` entry to record the update. This can be useful to determine if history-related bugs apply to your cluster. The entry can also reveal correlation between changes in cluster behavior with potential triggers, although correlation does not imply causation. 
```

Comment:
```suggestion
When updating your cluster, the cluster version Operator (CVO) creates a `status.history` entry to record the update. This can be useful to determine if history-related bugs apply to your cluster. The entry can also reveal correlation between changes in cluster behavior with potential triggers, although correlation does not imply causation. 
```
"Operator" should be capitalized - I also wonder, should it be "Cluster Version Operator" if that's an official name?

Suggestion: It may be worth adding an attribute for the Operator name if one doesn't already exist.

Reply:
Would it be helpful to users to mention here where the "`status.history` entry" is created? Is this part of some system logs, etc?

Reply:
good call - I'll confirm with the engineers :)

Reply:
I've removed that sentence since it was more confusing than accurate. :) thanks!